### DISCUSSION TITLE
Question about webrtc.NewTrackLocalStaticRTP

### DISCUSSION THREAD

**Discussion by rwschnetler** [pion/webrtc#1547] [General] [answered]
Hi, Obviously doing something stupid, but if anybody can help me, it will be highly appreciated:

Trying to compile the example rtp-to-webrtc as is, and get the following error:

./main.go:53:21: undefined: webrtc.NewTrackLocalStaticRTP

The code is exactly as in the example but here is the relevant part:

package main

_import (
	"fmt"
	"net"

	"github.com/pion/rtp"
	"github.com/pion/webrtc/v3"
	"projects/wilmar_rtmp/internal/signal"
)

func main() {
.
.
.
	// Create a video track
	videoTrack, err := **webrtc.NewTrackLocalStaticRTP**(webrtc.RTPCodecCapability{MimeType: "video/vp8"}, "video", "pion")
	if err != nil {
		panic(err)
	}
	if _, err = peerConnection.AddTrack(videoTrack); err != nil {
		panic(err)
	}
.
.
.
}_

**Comment by rwschnetler** [comment]
Also 

the last line videoTrack.Write fails too:

The error is _'Unresolved reference'_

_if _, writeErr := videoTrack.**Write**(inboundRTPPacket[:n]); writeErr != nil {
			panic(writeErr)
		}_

**Comment by Sean-Der** [comment, marked as answer]
Hey @rwschnetler 

What version of pion/webrtc are you using? Older versions of v3.0.0 beta didnt have this function yet. Thanks!
